README.md

```diff
@@ -144,15 +144,15 @@ behaviour. Possible choices with this switch are:
 *replaygain.org* (alias: *fb2k*)
   Replay Gain information is stored in ID3v2 TXXX frames. This format is
   specified on the replaygain.org website as the recommended format for MP3
-  files. Notably, this format is also used by the [foobar2000 music player for
-  Windows][4].
+   files. Notably, this format is used by music players like [foobar2000][4] 
+   and [Quod Libet][5]. The latter can also fall back on the legacy format.
```

Comment:
Would you mind also updating the link to the Quod Libet repository in this file (L211)?

Reply:
Happy to do it! :)